src/App.css

```diff
@@ -15,14 +15,14 @@
 
```

Comment:
just as an FYI the tab name in the browser for your app still says rare treasures?